HACKING.md

```diff
@@ -79,13 +79,13 @@ and yq (needed for yaml manipulation), download the newest image and import it i
 ```
 sudo snap install lxd
 sudo snap install yq
-curl -o lxd-initrd-img.tar.gz https://storage.googleapis.com/snapd-spread-core/lxd/lxd-spread-initrd22-img.tar.gz
-lxc image import lxd-initrd-img.tar.gz --alias ucspread22
+curl -o lxd-initrd-img.tar.gz https://storage.googleapis.com/snapd-spread-core/lxd/lxd-spread-initrd-img.tar.gz
+lxc image import lxd-spread-initrd-img.tar.gz --alias ucspread22
```

Comment:
This seems wrong as curl saves the output as `lxd-initrd-img.tar.gz` if you look at the `-o`